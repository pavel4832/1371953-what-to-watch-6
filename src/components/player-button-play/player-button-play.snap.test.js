import React from 'react';
import {Provider} from 'react-redux';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import PlayerButtonPlay from './player-button-play';
import configureStore from 'redux-mock-store';

const mockStore = configureStore({});

it(`Should PlayerButtonPlay render correctly`, () => {
  const history = createMemoryHistory();

  const {container} = render(
      <Provider store={mockStore({})}>
        <Router history={history}>
          <PlayerButtonPlay isPlay={false} onButtonClick={jest.fn()} />
        </Router>
      </Provider>
  );

  expect(container).toMatchSnapshot();
});
