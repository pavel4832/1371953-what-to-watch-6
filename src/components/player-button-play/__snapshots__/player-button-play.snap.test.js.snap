// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PlayerButtonPlay render correctly 1`] = `
<div>
  <button
    class="player__play"
    type="button"
  >
    <svg
      height="19"
      viewBox="0 0 19 19"
      width="19"
    >
      <use
        xlink:href="#play-s"
      />
    </svg>
    <span>
      Play
    </span>
  </button>
</div>
`;
